### DISCUSSION TITLE
How to convert yolox to tflite int8?

### DISCUSSION THREAD

**Discussion by Alter-Y** [Megvii-BaseDetection/YOLOX#1322] [General]
I transformed the weights of yolox from torch to tensorflow keras, and now i want to transform keras model to tflite.  But when using int8 quantization, the output array of confidence and class ([1, 8500, 5:]) are all 0. I found it's because  yolox performed convolution on box, confidence, and class, respectively, in yolo_head.py. And box outputs are not activated with sigmoid, it causes problem  for concatenation layers when the input ranges have a great difference. So, how to fix this problem?
